docs/EditContext/explainer.md

```diff
@@ -251,7 +251,7 @@ The following table summarizes the difference between div with contentEditable a
 ```
 
 ## Example Application
-This [example](canvas_editContext.html) shows how an author can use EditContext to implement (IME) typing on a &lt;canvas&gt; element. ([demo video](https://microsoft-my.sharepoint-df.com/:v:/g/personal/shihken_microsoft_com1/EZxVZX0o4XhIqUj6lXuY-soB2rbXNBRMtNPx1jU40rmpOA))
+This [example](canvas_editContext.html) shows how an author can use EditContext to implement (IME) typing on a &lt;canvas&gt; element. ([demo video](https://alexkeng.github.io/editing/docs/EditContext/EditContext_tpac2021.mp4))
```

Comment:
I see the video is also uploaded to GitHub in this PR. Probably need to change the link to w3c.github.io/ after merging this.

Reply:
good catch! I'll remove the absolute path